developer-tools.md

```diff
@@ -463,25 +463,16 @@ in the Eclipse install directory. Increase the following setting as needed:
 <a name="nightly-builds"></a>
 <h3>Nightly Builds</h3>
 
-Packages are built regularly off of Spark's master branch and release branches. These provide 
-Spark developers access to the bleeding-edge of Spark master or the most recent fixes not yet 
-incorporated into a maintenance release. These should only be used by Spark developers, as they 
-may have bugs and have not undergone the same level of testing as releases. Spark nightly packages 
-are available at:
-
-- Latest master build: <a href="https://people.apache.org/~pwendell/spark-nightly/spark-master-bin/latest">https://people.apache.org/~pwendell/spark-nightly/spark-master-bin/latest</a>
-- All nightly builds: <a href="https://people.apache.org/~pwendell/spark-nightly/">https://people.apache.org/~pwendell/spark-nightly/</a>
-
-Spark also publishes SNAPSHOT releases of its Maven artifacts for both master and maintenance 
+Spark publishes SNAPSHOT releases of its Maven artifacts for both master and maintenance 
```

Comment:
We don't publish nightly builds anymore.

Reply:
and maybe there's no point to publish SNAPSHOT artifacts? ...